extension_templates/forecasting.py

```diff
@@ -261,6 +264,31 @@ def _predict_quantiles(self, fh, X=None, alpha=0.5):
         """
         # implement here
 
+    def _predict_var(self, fh, X=None, cov=False):
+        """
+        Compute/return prediction variance for a forecast.
+
+        Must be run *after* the forecaster has been fitted.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
```

Comment:
when it shows up in `_predict_var`, it´s *always* a `ForecastingHorizon` - compare `_predict` docstring

Reply:
Understood, the `predict` method in `BaseForecaster` does the check/traformation of `fh`